operations/network/management_network/aruba/mclag_isl_ha.md

```diff
@@ -4,12 +4,12 @@ The intent here is to show case a ISL (inter-switch-link) link failover scenario
 
 In below a visualization of disconnected ISL link:
 
-![](../img/vsx_isl_ha.png)
+![](../../../../../../../img/network/management_network/network/management_network/vsx_isl_ha.png)
```

Comment:
Seems to be duplicate amount of "../" and "network/management_network"

Reply:
fixed